homeassistant/helpers/start.py

```diff
@@ -24,4 +24,4 @@ async def _matched_event(event: Event) -> None:
         """Call the callback when Home Assistant started."""
         hass.async_run_hass_job(at_start_job, hass)
 
-    return hass.bus.async_listen_once(EVENT_HOMEASSISTANT_START, _matched_event)
+    return hass.bus.async_listen_once(EVENT_HOMEASSISTANT_START, _matched_event).cancel
```

Comment:
It's backward compatible, I guess we should just return the object?

Reply:
It's backwards compatible, but it breaks typing if we return the object.